ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -457,12 +497,17 @@ function ListRules(props) {
       filteredRules = filteredRules.filter(rule => rule.scheduledChange);
     }
 
+    const rewoundFilteredRules = clone(rewoundRules);
+    // use this line for non-diff mode on rewound rules
+    let rulesToShow =
+      rewoundRules.length === 0 ? filteredRules : rewoundFilteredRules;
```

Comment:
issue: This conditional is close, but misses the edge case of rewinding far enough in the past that no rules existed. Eg: if I rewind back to 2012, I see the current rules instead of no rules at all.

a related suggestion (non-blocking): When there are no rules to show, it might be good to show some text to confirm to the user that there are no rules matching the rewind and/or product/channel filter. (Otherwise the user may assume something has gone wrong.)